code/mob/living/critter/small_animal.dm

```diff
@@ -522,19 +524,6 @@ proc/filter_carrier_pets(var/type)
 				src.visible_message(SPAN_NOTICE("[src] notices a Clown and knocks [human] over!"))
 				human.setStatus("resting", duration = INFINITE_STATUS)
 
-	seek_target(range)
-		. = list()
-		var/list/hearers_list = hearers(range, src)
-		for (var/mob/living/M in hearers_list)
-			if (istype(M, /mob/living/carbon/human))
-				var/mob/living/carbon/human/human = M
-				var/clown_tally = human.clown_tally()
-				if (isalive(human) && (clown_tally>=2 || human.traitHolder.hasTrait("training_clown"))) //We only hate clowns
-					. += human
-		if (length(.))
-			if (!ON_COOLDOWN(src,"clown_charge_alert", 2 MINUTES))
-				src.visible_message(SPAN_ALERT("<b>[src]</b> notices a Clown and starts charging!"))
-
```

Comment:
Just removing this has made normal turtles aggressive to everyone, you'll need to change their behaviour to be retaliate-only as well.